app/components/candidate_interface/pending_gcse_required_component.rb

```diff
@@ -0,0 +1,25 @@
+class CandidateInterface::PendingGcseRequiredComponent < ViewComponent::Base
+  attr_accessor :application_choice, :pending_uk_gcses
+
+  def initialize(application_choice)
+    @application_choice = application_choice
+    @pending_uk_gcses = find_pending_uk_gcses(application_choice)
+  end
+
+  def pending_gcse_and_course_can_accept?
+    @pending_uk_gcses.any? && !@application_choice.course.accept_pending_gcse?
+  end
+
+private
+
+  def pending_gcse_text
+    subjects = pending_uk_gcses.map(&:subject)
+    subjects[0] = subjects[0].capitalize
```

Comment:
Is this assuming that the first pending UK GCSE will always be English. Say the candidate only has a pending GCSE in maths won't this output 

```
You said you’re currently studying for a qualification in Maths
```

maybe something like 

```
subjects.map { |subject| subject.capitalize if subject == 'english' }
```

Reply:
I think I was going for the first of any subject, as it could feasibly be a combination of all 3 and so 'English' might not be relevant. 

This way the formatting should apply universally? what do you reckon? Whilst munging the data the order was not always consistent as iIthink it orders them by `updated at`

Reply:
Might be worth chatting to Binisa or Emma, but i'm 99% sure with maths, English and science, science and maths are always lower case (mid sentence) and English is always capitalised.

Reply:
Just spoke with Emma and you're 100% right, will fix.